scripts/averaging/create_TEM_files.py

```diff
@@ -136,53 +174,75 @@ def create_TEM_files(adf):
         if not starting_location.is_dir():
             emsg = f"Provided 'cam_hist_loc' directory '{starting_location}' not found."
             emsg += " Script is ending here."
-            adf.end_diag_fail(emsg)
+            return
```

Comment:
I think we do want the `end_diag_fail` here, because if `cam_hist_locs`  can't be found then something has gone catastrophically wrong, and we want to kill the ADF as soon as possible.